plugins/transport-nio/src/test/java/org/elasticsearch/http/nio/NioHttpServerTransportTests.java

```diff
@@ -370,4 +377,8 @@ public void dispatchBadRequest(final RestChannel channel,
             }
         }
     }
+
+    private String requestToString(RestRequest restRequest) {
```

Comment:
I think this can be `static` and moved to somewhere in the test framework module?